### PR TITLE
Copy text on click

### LINKED ISSUE #22: Favourited cards do not show expiration date and cvc
Hello,

First of all, thanks for the extension. I use it everyday and helps me a lot.

This issue is about the favourite feature. I do not have the adyen drop-in payment flow so I do not benefit from the `Prefill` button, so I manually copy the card number, cvc and expiration date to test my checkout. I currently can not use favourites because when I do the cvc and expiration date are replaced by the brand logo, so I have no way of copying the CVC and EXP date. 

**My suggestion**: Keep the CVC and expiration date and make it so when you click at the field you copy the value. You can still keep the brand logo at the end of everything maybe.
I will be happy to submit a PR if you like this idea, let me know :)

Thanks again.

### PR DESCRIPTION
**Pull request by gcatanese**
This PR reworks the `copy-to-clipboard` feature:

- The copy link is removed
- Each copiable element (card number, expiry date, CVC, IBAN, etc..) is copied by clicking on the text
- A fixed (always visible when scrolling) header displays a message when an item is clicked and copied into the clipboard
- The Favourites section includes extra information (expiry date, CVC, IBAN holder), these fields can also be copied into the clipboard 

Fix #22

### CODE REVIEW

File: data/giftcards.json [hunk 1]
```diff
@@ -1,25 +1,25 @@
 [
     {
         "cardnumber" : "6036280000000000000",
-        "code" : "ANY",
```
Comment (Kwok-he-Chu) [inline data/giftcards.json:4]: Change this to '100' as default value, see: 

https://docs.adyen.com/payment-methods/gift-cards/web-drop-in/#test-and-go-live


| Amount (last three digits) | Result Code | Refusal Reason           |
|---------------------------|-------------|--------------------------|
| 100                       | Authorised  |                          |
| 123                       | Refused     | Refused                  |
| 124                       | Refused     | Not enough balance       |
| 125                       | Refused     | Blocked Card             |
| 126                       | Refused     | Expired Card             |
| 130                       | Error       | Acquirer Error           |
| 134                       | Refused     | Invalid Pin              |
| 135                       | Refused     | Pin tries exceeded       |

File: e2e/tests/test-extension.spec.ts [hunk 2]
```diff
@@ -2,38 +2,110 @@ import { test, expect } from './fixtures';
...
+    // card number
+    let cardnumber = page.locator('text="6036280000000000000"');
+    await expect(cardnumber).toBeVisible();
+    await cardnumber.click();
+
+    let clipboard = await page.evaluate("navigator.clipboard.readText()");
+    expect(clipboard).toContain("6036280000000000000");
+
+    // code
+    let code = page.locator('text="123"').first();
```
Comment (Kwok-he-Chu) [inline e2e/tests/test-extension.spec.ts:82]: check

Comment (Kwok-he-Chu) [reply e2e/tests/test-extension.spec.ts:82]: - [ ] Add workflow/github action that runs test

File: data/giftcards.json (hunk 1 above)
Comment (gcatanese) [reply data/giftcards.json:4]: Good catch

File: e2e/tests/test-extension.spec.ts (hunk 2 above)
Comment (gcatanese) [reply e2e/tests/test-extension.spec.ts]: The GitHub action will be created as part of #30

**Pull request merged**